engine/glfw/lib/ios/app/ViewController.m

```diff
@@ -261,6 +261,10 @@ void _glfwPlatformSetViewType(int view_type)
     g_view_type = view_type;
 }
 
+void _glfwPlatformSetWindowBackgroundColor(int color)
+{
+}
+
```

Comment:
hmm, meybe we an change color here as well? https://github.com/defold/defold/issues/6142

Reply:
Yes, absolutely! I've so far only focused on Windows in this PR

Reply:
color + colour :)